support-frontend/cloud-formation/cfn.yaml

```diff
@@ -503,3 +505,18 @@ Resources:
       TreatMissingData: notBreaching
     DependsOn:
     - StateMachineUnavailableMetric
+
+  S3Bucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      AccessControl: PublicRead
+      BucketName: !Join 
+      - '-'
+      - - !Ref Stack
+        - !FindInMap [ StageVariables, !Ref Stage, BucketSuffix ]
+        - !Ref App
+        - static
+      MetricsConfigurations: 
+        - Id: EntireBucket
```

Comment:
What is MetricsConfiguration for?

Reply:
That seems to be so we can tell whether we are getting requests sucessfully, we forgot to check what it actually sends though so good spot
Docs are here https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket-metricsconfiguration.html

Reply:
![image](https://user-images.githubusercontent.com/7304387/61135872-6496fd80-a4ba-11e9-8129-03ac6ffa4108.png)